applications/serial_lte_modem/doc/PPP_AT_commands.rst

```diff
@@ -93,3 +95,42 @@ Example
   #XPPP: 0,0
 
   OK
+
+Testing on Linux
+================
+
+You can test SLM's PPP on Linux by using the ``pppd`` command, though SLM must be compiled without CMUX because there is no widely available utility that allows multiplexing a device file on Linux.
+
+.. note::
+
+   If you have a utility that allows multiplexing a device file on Linux, you can use SLM's PPP with the ``pppd`` command through CMUX.
+   To do this, you must first set up the CMUX link.
+   Then, make sure to replace the device file argument in the ``pppd`` command with that of SLM's PPP channel, which will have been created by the CMUX utility.
+   See :ref:`SLM_AT_CMUX` for more information on SLM's CMUX.
+
+For the process described here, SLM's UARTs must be connected to the Linux host.
+
+1. Get PPP running on SLM.
+   To do this, start SLM and issue an ``AT+CFUN=1`` command.
+
+#. Make sure that the network registration succeeds and that PPP is started successfully.
+   To do this, either look at SLM's logs, or issue an ``AT#XPPP?`` command, which returns ``#XPPP: 1,0`` when PPP has started successfully.
+
+#. Run the following command on the Linux host:
+
+   .. code-block:: console
+
+      $ sudo pppd -detach <PPP_UART_dev> <baud_rate> noauth crtscts novj nodeflate nobsdcomp debug +ipv6 usepeerdns noipdefault defaultroute defaultroute6 ipv6cp-restart 5 ipcp-restart 5
+
+   Replace ``<PPP_UART_dev>`` by the device file assigned to the PPP UART and ``<baud_rate>`` by the baud rate of the UART that PPP is using.
+   Typically, when ``uart1`` is assigned to be the PPP UART (in the devicetree overlay), the device file assigned to it is :file:`/dev/ttyACM2` for an nRF9160 DK, and :file:`/dev/ttyACM2` for the other nRF91 Series DKs.
```

Comment:
Second /dev/ttyACM2 -> /dev/ttyACM1.

Also, how about mentioning the typical 1000 000 baud rate in here? Maybe better in here than in the configurations.

This is already merged, so fix at least the first one in the PR that is open.

Reply:
Well spotted.